packages/react-components/src/organisms/ResultList.tsx

```diff
@@ -15,7 +15,18 @@ const headerStyles = css({
   display: 'grid',
```

Comment:
make this a flex-wrap for auto wrapping, will be much simpler and will know when to wrap more exactly than a breakpoint 🙃